docker/buildkite/grafana/provisioning/dashboards/cadence-client-overall.json

```diff
@@ -392,7 +391,32 @@
             ]
           }
         },
-        "overrides": []
+        "overrides": [
+          {
+            "__systemRef": "hideSeriesFrom",
+            "matcher": {
+              "id": "byNames",
+              "options": {
+                "mode": "exclude",
+                "names": [
+                  "HistoryClientRespondDecisionTaskCompleted"
```

Comment:
often worth excluding poll-responses since they generally don't have issues like this and they just retry.  but an exclude by regex for `respond(decision|activity)` is probably best in that case.